src/Pages/signup/Signup.jsx

```diff
@@ -0,0 +1,151 @@
+import "./Signup.css";
+import { useAuth } from "../../Contexts";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+const Signup = () => {
+  const navigate = useNavigate();
+  const { signupHandler, token } = useAuth();
+  const [signUpDetails, setSignUpDetails] = useState({
+    email: "",
+    password: "",
+    firstname: "",
+    lastname: "",
+    confirmpassword: "",
+  });
+
+  useEffect(() => {
+    if (token) {
+      setTimeout(() => {
+        navigate("/home");
+      }, 1000);
+    }
+  }, [token]);
+  return (
+    <>
+      {" "}
+      <div className="signup-page">
+        <div className="login-container">
+          <div className="login-inputs login-inputs-signup">
+            <h1>Sign Up</h1>
+            <form
```

Comment:
You can give toast message for error handling on form submission for sign in and sign up pages

Reply:
I need to learn toastify . I will implement after that